projects/plugins/jetpack/extensions/blocks/videopress/editor.js

```diff
@@ -114,6 +138,17 @@ const addVideoPressSupport = ( settings, name ) => {
 			`jetpack/videopress-with-has-warning-is-interactive-class-names`,
 			withHasWarningIsInteractiveClassNames( `core/video` )
 		);
+	} else if ( available ) {
+		// If VideoPress is available, we update the block description and example with VideoPress-specific content.
+		settings.description = __(
+			'Embed a video from your media library or upload a new one with VideoPress.',
+			'jetpack'
+		);
+		settings.example.attributes = {
+			caption: __( 'Upload or embed videos with VideoPress', 'jetpack' ),
+			isVideoPressExample: true,
+			src: 'https://videopress2.files.wordpress.com/2021/11/videopress-devices.jpg',
```

Comment:
In the past we've uploaded them to the plugin, so they can be served locally. Here is an example:
https://github.com/Automattic/jetpack/blob/2704efac24c831348233c68f6ab35893d706ddf7/projects/plugins/jetpack/extensions/blocks/tiled-gallery/index.js#L32-L40

Should we do that here too?

Reply:
Sure, I'll update the PR ! :)